index.html

```diff
@@ -257,10 +264,16 @@ <h2 class="contact-description dark-blue">
             class="change"
             required
           ></textarea>
-          <img src="img/contact-form-png.svg" alt="just background png" />
+          <img
+            src="img/contact-form-png.svg"
+            alt="just background png"
+            class="only-mobile"
+          />
           <button type="submit" class="form-btn">Get in touch</button>
         </form>
+        <div class="contact-png"></div>
```

Comment:
- [ ] Well done creating the desktop version for the contact form, let's make it even better like the Figma template:

Template:
![Captura de pantalla 2022-03-25 a la(s) 11 16 21 p  m](https://user-images.githubusercontent.com/83717285/160224655-3db51e77-058a-433e-8661-7a0189ca1309.png)

Your design:
![Captura de pantalla 2022-03-25 a la(s) 11 25 20 p  m](https://user-images.githubusercontent.com/83717285/160224660-c7514596-df6c-480b-872b-0a48c967b26c.png)

Try justifying the text and reducing the vertical size of the whole section.

Reply:
Done, thank you for the observation.

Reply:
partially resolved